contrib/debian/copyright

```diff
@@ -1158,9 +1261,6 @@ License: GPL-3+ with Autoconf exception
  You should have received a copy of the GNU General Public License along
  with this program. If not, see <http://www.gnu.org/licenses/>.
  .
- On Debian systems, the full text of the GNU General Public License
- version 3 can be found in the file `/usr/share/common-licenses/GPL-3'.
- .
```

Comment:
NACK. I added this in https://github.com/zcash/zcash/pull/5333/commits/a672318b4d9341a2ac3938f2327a3f218ad88ae8, again to follow the Debian copyright file v1 format.

Reply:
Removed this commit.